mobile/android/app/google-services.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
🛠️ Afaik we are using a dynamic approach for this google-service.json because we need to support our dev env and mainnet.
I'm happy to take this over and test locally..